prow/jobs/modules/internal/serverless-manager.yaml

```diff
@@ -599,7 +599,7 @@ presubmits: # runs on PRs
               - "sh"
             args:
               - "-c"
-              - "cp -r config/serverless/ module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile"
+              - "mkdir -p module-chart && cp -r config/serverless/* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile"
```

Comment:
```suggestion
              - "mkdir -p module-chart && cp -r config/serverless/\* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile"
```

Reply:
Why? This is shell command - not regexp.